liquibase-standard/src/main/java/liquibase/command/core/AbstractUpdateCommandStep.java

```diff
@@ -89,6 +89,8 @@ public void run(CommandResultsBuilder resultsBuilder) throws Exception {
                 ShowSummaryUtil.showUpdateSummary(databaseChangeLog, getShowSummary(commandScope), statusVisitor, resultsBuilder.getOutputStream());
             });
 
+            database.afterUpdate();
```

Comment:
@StevenMassaro what do you think about this solution here? I"ve been thinking of options to handle this "afterUpdate" but I'm out of ideas.

Reply:
What about putting this in the ChangeExecListener class? There is already a `ran` method there that you could key into.

Reply:
A `ChangeExecListener` will be invoked *per Changeset* by definition...
>An implementation of ChangeExecListener can be called by UpdateVisitor or RollbackVisitor for each changeset that is actually run.

 ...so it would be wrong to add an event handling method to it that is run after *the last* Changeset.

Reply:
A solution could be to rephrase the JavaDoc of `ChangeExecListener`. @filipelautert How to proceed?

Reply:
@mkarg So I tried to get a working implementation using the ChangeExecListener approach but it was becoming a big refactoring to accommodate a simple update like this one = it is really meant to be executed for each changeset as the javadoc says, cause ChangeExecListener is called by UpdateVisitor that is called by changelogIterator and so on.
So let's go with your solution !
I just did a small refactoring to use the already existing sql executor method instead of creating a new one - could you confirm if it's fine for you?